### ISSUE TITLE
bower:clean fails without Bowerfile

### ISSUE THREAD

**Issue by leschenko** [rharriso/bower-rails#175]
`rake bower:clean` fails without Bowerfile

Attempting to remove all but main files as specified by bower
rake aborted!
Errno::ENOENT: No such file or directory @ rb_sysopen - my_project_path/Bowerfile
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/dsl.rb:70:in `initialize'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/dsl.rb:70:in`open'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/dsl.rb:70:in `eval_file'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/dsl.rb:10:in`block in evalute'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/dsl.rb:10:in `tap'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/dsl.rb:10:in`evalute'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/performer.rb:44:in `dsl'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/performer.rb:199:in`main_files_for_component'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/performer.rb:162:in `block in remove_extra_files'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/performer.rb:149:in`each'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/performer.rb:149:in `remove_extra_files'
my_rvm_path/gems/bower-rails-0.10.0/lib/tasks/bower.rake:74:in`block (3 levels) in <top (required)>'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/performer.rb:36:in `instance_exec'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/performer.rb:36:in`block in perform'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/performer.rb:107:in `block (2 levels) in perform_command'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/performer.rb:91:in`chdir'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/performer.rb:91:in `block in perform_command'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/performer.rb:81:in`each'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/performer.rb:81:in `perform_command'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/performer.rb:35:in`perform'
my_rvm_path/gems/bower-rails-0.10.0/lib/bower-rails/performer.rb:10:in `perform'
my_rvm_path/gems/bower-rails-0.10.0/lib/tasks/bower.rake:73:in`block (2 levels) in <top (required)>'
my_rvm_path/bin/ruby_executable_hooks:15:in `eval'
my_rvm_path/bin/ruby_executable_hooks:15:in`<main>'
Tasks: TOP => bower:clean
(See full trace by running task with --trace)

Here is a workaround

``` ruby
BowerRails::Performer.class_eval do
  def main_files_for_component(name)
    []
  end
end
```

**Comment by JakeLaCombe** [comment]
I'm getting this issue to when running a deployment on Heroku.

**Comment by tejasbubane** [comment]
Getting the same issue.

```shell
⇒  rake bower:clean

Attempting to remove all but main files as specified by bower
rake aborted!
Errno::ENOENT: No such file or directory @ rb_sysopen - /Users/tejas/<path-to-my-project>/Bowerfile
```

```shell
⇒  bower --version
1.7.7
```

```shell
⇒  bundle exec gem list | grep bower
bower-rails (0.10.0)
```

**Comment by inlikealion** [comment]
I'm getting this, too.

```
⇒ rake bower:clean

Attempting to remove all but main files as specified by bower
rake aborted!
Errno::ENOENT: No such file or directory @ rb_sysopen - /Users/baintonm/Documents/Sites/ui.toolinaround.com/Bowerfile
```

```
⇒ bower --version
1.7.9
```

```
⇒ bundle exec gem list | grep bower
bower-rails (0.10.0)
```

**Comment by chaitalipatel** [comment]
Any luck sorting out this issue?

**Comment by SergeyKishenin** [comment]
@chaitalipatel Have you tried `v0.11.0`. Seems like it was fixed in #175
